test/browser/fragments.test.js

```diff
@@ -568,7 +568,7 @@ describe('Fragment', () => {
 		expect(scratch.innerHTML).to.equal('<div>Hello</div>');
 	});
 
-	it.skip('should preserve state between double nested fragment and double nested array', () => {
```

Comment:
This test was skipped in the world where we flattened arrays. Since we now convert arrays to Fragments it passes. Went ahead and enabled it.